src/VsixV3/Microsoft.Net.Sdk.swr

```diff
@@ -0,0 +1,32 @@
+use vs
+
+package name=Microsoft.Net.Sdk
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft Net Sdk"
+                       description="SDK for .NET projects"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\sdk"
+  file source="$(OutputPath)sdk\Sdk.props"
+  file source="$(OutputPath)sdk\Sdk.targets"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\build"
```

Comment:
We also need `buildCrossTargeting\Microsoft.NET.Sdk.targets`.  The cross targeting `.props` file is no longer needed, but the cross targeting `.targets` file is [imported in Sdk.targets](https://github.com/dotnet/sdk/blob/88733a44ea2ea464c9f06462c80714c4cc3250a2/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets#L37-L38)